### PR TITLE
Correctly set and restore the TLV and remove the Cargo feature for it

### PR DESCRIPTION
**Pull request by Zoxc**
For some reason I didn't think to restore the TLV after running other Rayon jobs.

r? @cuviper  / @nikomatsakis

### CODE REVIEW

File: rayon-core/Cargo.toml [hunk 1]
```diff
@@ -12,10 +12,6 @@ readme = "README.md"
 keywords = ["parallel", "thread", "concurrency", "join", "performance"]
 categories = ["concurrency"]
 
-[features]
-default = ["tlv"]
-tlv = []
```
Comment (eddyb) [inline rayon-core/Cargo.toml]: Isn't removing a feature backwards-incompatible?

Comment (Zoxc) [reply rayon-core/Cargo.toml]: Probably, `rustc-rayon` doesn't provide any guarantees though.

Comment (nikomatsakis) [reply rayon-core/Cargo.toml]: Still, might as well bump to "0.2.0" I would think?

Comment (nikomatsakis) [reply rayon-core/Cargo.toml]: (At least that's what I usually do)

**Review by nikomatsakis — APPROVED** [review]

**Comment by nikomatsakis** [comment]
I'm 👍 to merge but would prefer to set version to `0.2.0` first

**Comment by Zoxc** [comment]
Merged and published `0.2.0`.

**Pull request closed without merging**